tests/unit/test_api_diagnostics.py

```
@@ -1383,3 +1383,196 @@ def test_ciissversion_validated_and_declared_keys_present(self):
+
+# ============================================================================
+# Option A: friendlier display name for incomplete-AROI fallback rows
+# ============================================================================
+
+
+class TestIncompleteAROIDisplayName:
+    """Option A: leaderboard rows for relays with a parseable url: token
+    but failed AROI parsing now render as '<domain> (incomplete AROI)'
+    instead of a truncated raw blob."""
+
+    def _name(self, contact, contact_hash='aaaaaaaa'):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_name`

(ANN202)